asf_welsh_energy_consultation/analysis/produce_plots_and_stats.py

```diff
@@ -32,11 +34,102 @@
     os.makedirs(output_folder)
 
 
+def create_argparser():
+    """
+    Creates an Argument Parser that can receive the following arguments:
+    - local_data_dir
+    - epc_batch
+    - mcs_batch
+
+    Returns:
+        Argument Parser
+    """
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--local_data_dir",
+        help="Local directory where EPC data is/will be stored",
+        type=str,
+    )
+
+    parser.add_argument(
+        "--supp_data",
+        help='Name of directory where supplementary data is stored in the form `data_YYYYMM`. Defaults to "newest"',
```

Comment:
perhaps just use subdirectory so that a future user doesn't confuse this with input_data_path
```suggestion
        help='Name of sub-directory where supplementary data is stored in the form `data_YYYYMM`. Defaults to "newest"',
```